modulefiles/ufs_common.lua

```diff
@@ -0,0 +1,32 @@
+whatis("Description: UFS build environment common libraries")
+
+help([[Load UFS Model common libraries]])
+
+local ufs_modules = {
+  {["jasper"]          = "2.0.32"},
+  {["zlib"]            = "1.2.13"},
+  {["libpng"]          = "1.6.37"},
+  {["hdf5"]            = "1.14.0"},
+  {["netcdf-c"]        = "4.9.2"},
+  {["netcdf-fortran"]  = "4.6.1"},
+  {["parallelio"]      = "2.5.10"},
+  {["esmf"]            = "8.6.0"},
+  {["fms"]             = "2024.01"},
+  {["bacio"]           = "2.4.1"},
+  {["crtm"]            = "2.4.0"},
+  {["g2"]              = "3.5.1"},
+  {["g2tmpl"]          = "1.13.0"},
+  {["ip"]              = "4.3.0"},
+  {["sp"]              = "2.5.0"},
+  {["w3emc"]           = "2.10.0"},
+  {["gftl-shared"]     = "1.6.1"},
+  {["mapl"]            = "2.40.3-esmf-8.6.0"},
+  {["scotch"]          = "7.0.4"},
```

Comment:
I would guess we could remove these?

Reply:
We probably could. At least some of them. Definitely don’t need many of these like scotch and w3emc. These are for the wave model